README.md

```diff
@@ -4,7 +4,7 @@ This is the official [Gel](https://github.com/geldata/gel) (formerly EdgeDB) cli
 for Dart.
 
 If you're just getting started with Gel, we recommend going through the
-[Gel Quickstart](https://www.geldata.com/p/quickstart-docs) first. This walks
+[Gel Quickstart](https://docs.geldata.com/learn/quickstart/overview) first. This walks
```

Comment:
https://www.geldata.com/p/quickstart-docs is already a redirect to https://docs.geldata.com/learn/quickstart/overview, and I think using the redirect is better so we don't have to update the link everywhere if we want to change the quickstart entry point in the future.

Reply:
OK, I've reverted back to the redirect links.